src/queries/dash/domain-list.sql

```diff
@@ -38,9 +38,6 @@ WITH pvs AS (
     AND hostname NOT LIKE '%.hlx.live'
     AND hostname NOT LIKE '%.helix3.dev'
     AND hostname NOT LIKE '%.sharepoint.com'
-    AND hostname NOT LIKE '%.google.com'
```

Comment:
Why not exclude google?  I would be concerned that this will increase our count of domains only related to authoring activity.

Reply:
right, understand. good point, re-adding